src/dmd/statementsem.d

```diff
@@ -2300,6 +2300,35 @@ else
                 }
             }
         }
+        else if (ps.ident == Id.linkerdirective)
+        {
+            version (all)
```

Comment:
Rather than using version all, remove the dead code.

Reply:
This is a copy from the code above... I have no idea why it's like that, but I felt I should respect precedent.

Reply:
Then perhaps rather than copy pasting code around, you should commonize it to a function.

Reply:
It didn't feel like enough code to warrant that treatment... are you asking me to do that?